src/main.py

```diff
@@ -15,4 +15,64 @@ def main():
 
 
 if __name__ == "__main__":
+    main()"""
+
+
+import sys
+import os
+from reader.reader import reader_mode
+from admin.admin import admin_mode
+from data.data_handler import GRAPH_FOLDER
+from data.graph_handler import admin_plot, reader_plot
```

Comment:
A saját modulokhoz használj relatív importot, vagy írd eléjük, hogy `src.`
```suggestion
from .reader.reader import reader_mode
from .admin.admin import admin_mode
from .data.data_handler import GRAPH_FOLDER
from .data.graph_handler import admin_plot, reader_plot
```

Reply:
Köszönöm, Olivér! Kijavítom.